modules/lang-groovy/src/test/java/org/elasticsearch/messy/tests/BucketSelectorTests.java

```diff
@@ -93,8 +94,8 @@ public void setupSuiteScopeCluster() throws Exception {
         builders.add(client().prepareIndex("idx_with_gaps", "type").setSource(newDocBuilder(3, 1, 0, 0)));
         builders.add(client().prepareIndex("idx_with_gaps", "type").setSource(newDocBuilder(3, 3, 0, 0)));
 
-        client().preparePutIndexedScript().setId("my_script").setScriptLang(GroovyScriptEngineService.NAME)
-                .setSource("{ \"script\": \"Double.isNaN(_value0) ? false : (_value0 + _value1 > 100)\" }").get();
+        client().admin().cluster().preparePutStoredScript().setId("my_script").setScriptLang(GroovyScriptEngineService.NAME)
+                .setSource(new BytesArray("{ \"script\": \"Double.isNaN(_value0) ? false : (_value0 + _value1 > 100)\" }")).get();
```

Comment:
If we have changed the API here so you need a BytesReference instead of a String inthe PUT stored script API maybe we should meantion it in the breaking changes doc?